src/js/components/Home/Home.scss

```diff
@@ -1,3 +1,25 @@
 .home {
     background-color: #ccc;
+}
+.background-enter {
```

Comment:
white spaces between declarations and empty line at the end of the line

Reply:
fixed thanks.